lib/gl_rubocop/gl_cops/unique_identifier.rb

```diff
@@ -0,0 +1,25 @@
+module GLRubocop
+  module GLCops
+    class UniqueIdentifier < RuboCop::Cop::Base
+      MSG = 'View components must include a data-test-id attribute'.freeze
+      EMPTY_MSG = 'data-test-id attribute must not be empty'.freeze
+      UNIQUE_IDENTIFIER = 'data-test-id'.freeze
+
+      # rubocop:disable Metrics/AbcSize
+      def on_send(node)
+        return unless %i[render template].include?(node.method_name)
+        return unless File.exist?(processed_source.file_path)
```

Comment:
Could these two checks be refactored into a separate private method so that we don't have to disable the `Metrics/AbcSize` cop? I'm kinda wary of introducing the pattern of disabling cops for new code that we write